data/json/monsters/mammal.json

```
@@ -2099,6 +2099,45 @@
     "special_attacks": [ [ "EAT_CROP", 60 ] ],
     "flags": [ "SEES", "HEARS", "SMELLS", "PET_MOUNTABLE", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM" ]
   },
+  {
+    "id": "mon_mutant_moose",
+    "type": "MONSTER",
+    "name": { "str_sp": "great tusked moose" },
+    "description": "The great tusked moose is a mutant variety of megafauna that evolved after the Cataclysm.  Easily recognized by the underslung tusks off the side of each mouth, in addition to an impressive rack of horns.",
+    "default_faction": "herbivore",
+    "bodytype": "horse",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "386000 ml",
+    "weight": "462 kg",
+    "hp": 160,
+    "speed": 200,
+    "material": [ "flesh" ],
+    "symbol": "M",
+    "color": "brown_red",
+    "aggression": 5,
+    "morale": 120,
+    "aggro_character": false,
+    "melee_skill": 6,
+    "melee_dice": 5,
+    "melee_dice_sides": 4,
+    "melee_damage": [ { "damage_type": "cut", "amount": 5 } ],
+    "special_attacks": [ { "id": "smash", "throw_strength": 72 } ],
+    "dodge": 1,
+    "armor_bash": 14,
+    "armor_cut": 1,
+    "armor_bullet": 10,
+    "armor_elec": 1,
+    "vision_night": 7,
+    "path_settings": { "max_dist": 10 },
+    "anger_triggers": [ "HURT", "PLAYER_CLOSE" ],
+    "baby_flags": [ "SPRING" ],
```

```suggestion
    "anger_triggers": [ "HURT", "MATING_SEASON" ],
    "baby_flags": [ "AUTUMN" ],
```
To match the normal moose, and the comment below those lines that says "autumn is their mating season so baby_flags is defined so that they get angry then"

If you did mean to add `PLAYER_CLOSE`, maybe mention something in the description about these being more aggressive than the old-fashioned moose?

---

Adjusted the description and added back mating season